config-1_x_0.md

```diff
@@ -607,11 +599,10 @@ Exogress edge servers may perform content optimization.
 ### WebP
 
 WebP is an image format employing both lossy and lossless compression, along with animation and alpha transparency.
-Developed by Google, it is designed to create smaller or better looking images compared to the JPEG, PNG, or GIF image
+Developed by Google, it was designed to create smaller or better-looking images than the JPEG, PNG, or GIF image
```

Comment:
Funny, it's a copy-paste from wikipedia :)

Reply:
actually, this one is a matter of choice. both options work well